content/shared/influxdb3-get-started/setup.md

```diff
@@ -537,34 +535,54 @@ In your command, replace {{% code-placeholder-key %}}`YOUR_AUTH_TOKEN`{{% /code-
 Set the `INFLUXDB3_AUTH_TOKEN` environment variable to have the CLI use your
 token automatically:
 
-{{% code-placeholders "YOUR_AUTH_TOKEN" %}}
-```bash
+{{< tabs-wrapper >}}
```

Comment:
The tabs are now (incorrectly) nested. Separate into env var usage and CLI option usage.